frontend/src/components/ErrorBoundary/ErrorFallback.tsx

```
@@ -0,0 +1,176 @@
+          description: 'This page encountered an error and cannot be displayed.',
+        };
+      case 'feature':
+        return {
+          container: 'p-4',
+          card: 'bg-orange-50 border border-orange-200 rounded-lg p-4',
+          icon: '⚡',
+          title: 'Feature Unavailable',
+          description: 'This feature is temporarily unavailable.',
+        };
+      case 'component':
+      default:
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)